go/glacier/README.md

```diff
@@ -0,0 +1,74 @@
+# AWS SDK for Go code examples for Amazon Simple Storage Service Glacier
+
+## Purpose
+
+These examples demonstrate how to perform several Amazon S3 Glacier operations.
+
+## Prerequisites
+
+You must have an AWS account, and have your default credentials and AWS Region
+configured as described in
+[Configuring the AWS SDK for Go](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html)
+in the AWS SDK for Go Developer Guide.
+
+## Running the code
+
+### CreateVault/CreateVault.go
+
+This example creates an Amazon S3 Glacier vault.
+
+`go run CreateVault -v VAULT-NAME`
+
+- _VAULT-NAME_ is the name of the vault to create.
+
+The unit test mocks the service client and the `CreateVault` function.
+
+### UploadArchive/UploadArchive.go
+
+This example uploads a file to an Amazon S3 Glacier vault.
+
+`go run UploadArchive -v VAULT-NAME -f FILE-NAME`
+
+- _VAULT-NAME_ is the name of the vault to which the file is uploaded.
+- _FILE-NAME_ is the name of the file to upload.
+
+The unit test mocks the service client and the `UploadArchive` function.
+
+### Notes
+
+- We recommend that you grant this code least privilege,
+  or at most the minimum permissions required to perform the task.
+  For more information, see
+  [Grant Least Privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege)
+  in the AWS Identity and Access Management User Guide.
```

Comment:
*AWS Identity and Access Management User Guide*

Reply:
Fixed